src/CommandingOfficers/Ave.java

```diff
@@ -65,7 +65,7 @@
   public static final int SNOW_THRESHOLD = 100; // Big numbers for integer math.
   public static final int SNOW_PER_TURN = 400;
   public static final int SNOW_MELT_RATE = 100;
-  private int MAX_SNOW_SPREAD_RANGE = 2;
+  private int MAX_SNOW_SPREAD_RANGE = 1;
```

Comment:
Any particular reason for reducing her radii by 1?

Reply:
She snowballs too hard otherwise?

Reply:
I guess this doesn't need to be a balance patch, and I should probably acquire some actual data before nerfing. Besides, someone needs to hold Tech in check.

Reply:
You have also received a user request specifically to not nerf her, if I recall correctly, lol.

Reply:
Well, better not disappoint the fans then.